scripts/testacc_prepare_env.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 export PATH=$PATH:.
 
-argocd_version=${ARGOCD_VERSION:-v2.11.9}
+argocd_version=${ARGOCD_VERSION:-v2.12.7}
```

Comment:
A minor nit, otherwise LGTM

```suggestion
argocd_version=${ARGOCD_VERSION:-v2.12.8}
```